test/common/stream_info/stream_info_impl_test.cc

```diff
@@ -10,9 +10,11 @@
 #include "common/stream_info/stream_info_impl.h"
 
 #include "test/common/stream_info/test_int_accessor.h"
+#include "test/mocks/http/mocks.h"
```

Comment:
Why http mocks?

Reply:
This test used `Http::TestRequestHeaderMapImpl` from upstream/mocks.h before.
I thought it's included from http mocks. But actually they are defined in test_common/utility.h. Now it's been fixed